auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/service/AltusIAMService.java

```diff
@@ -40,6 +40,27 @@ public AltusIAMService(GrpcUmsClient umsClient,
                         UserManagementProto.AccessKeyType.Value.ED25519)));
     }
 
+    /**
+     * Checks that machine user has a specific access key on UMS side
+     */
+    public boolean doesMachineUserHasAccessKey(String machineUserName, String accessKey, String actorCrn, String accountId,
```

Comment:
It's quite confusing that you did not kept the order of variables but used the same name for the method! Somewhere the actorCrn is first, but here machineUserName is the first sine. Since we are only using strings here it's easy to misuse these methods.

Reply:
ok,right, im fixing it